src/components/FormElements/ImageUpload.js

```diff
@@ -0,0 +1,56 @@
+import React, { useRef, useState } from "react";
+import { Button } from "react-bootstrap";
+import "./ImageUpload.css";
+
+const ImageUpload = (props) => {
```

Comment:
Kindly make sure to add the proper comments, describing what is the purpose of this function, what it is accepting, and what it is returning